lib/jquery.expensifyIframify.js

```diff
@@ -217,7 +217,7 @@ export default {
                 return;
             }
 
-            if (!event.data || !_.isString(event.data)) {
+            if (!event.data || typeof event.data !== 'string') {
```

Comment:
According to grok these aren't 100% equivalent
```suggestion
            if (!event.data || (typeof event.data !== 'string' && !(event.data instanceof String))) {
```
<img width="824" alt="image" src="https://github.com/user-attachments/assets/025d7b99-8772-4078-bc32-74da2930d6c1" />